dump/dump.go

```diff
@@ -81,6 +81,10 @@ func Get(client *kong.Client, config Config) (*utils.KongRawState, error) {
 	}
 	state.Targets = targets
 
+	if config.SkipConsumers {
```

Comment:
nit: Please move this below line 52 to keep it near where the plugins are loaded.